config/moc.gypi

```diff
@@ -2,16 +2,17 @@
 # RUN: npm run automoc after updating moc.json
 {
     "sources": [
-        "../src/cpp/autogen/nwidget_moc.cpp",
-        "../src/cpp/autogen/nlabel_moc.cpp",
-        "../src/cpp/autogen/ncheckbox_moc.cpp",
-        "../src/cpp/autogen/nlineedit_moc.cpp",
-        "../src/cpp/autogen/nmainwindow_moc.cpp",
-        "../src/cpp/autogen/nprogressbar_moc.cpp",
-        "../src/cpp/autogen/npushbutton_moc.cpp",
-        "../src/cpp/autogen/nspinbox_moc.cpp",
-        "../src/cpp/autogen/nradiobutton_moc.cpp",
-        "../src/cpp/autogen/nplaintextedit_moc.cpp",
-        "../src/cpp/autogen/nscrollarea_moc.cpp"
+        "..\\src\\cpp\\autogen\\nwidget_moc.cpp",
```

Comment:
Ignore theese, its something I'll tacke in next days.
automoc.js is outputting paths in Windows format and we have to add a convertion layer on top to prevent that behaviour.